src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+(unreleased)
+++++++++++++++++++
+* vm: makes image create's '--source' argument mandatory, also improves the clarity of the help 
```

Comment:
Rec:
"BC: `image create`: `--source` is now a required argument"

You shouldn't need to call out updating help in the history file.